app/helpers/admin/product_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Admin::ProductHelper
+  def convert_param_to_date param, subkey
+    return param[subkey].to_date if
+      param.present? && param[subkey].present?
+
+    nil
```

Comment:
```suggestion
   param[subkey]&.to_date
```
như này được không em

Reply:
Cách này hay hơn ạ, để em sửa lại

Reply:
Để `param && param[subkey]&.to_date` mới đc ạ